transport/ice/IceSession.h

```diff
@@ -292,28 +307,25 @@ class IceSession
         uint64_t pendingRequestAge(uint64_t now) const;
         size_t pendingTransactionCount() const;
 
+        IceSession* _iceSession;
```

Comment:
Good change, but is there a case where this needs to be a pointer and not & ?

Reply:
Yes, I can change to a reference